analysis/val02_counters/counters01_prepare_data_nb.py

```diff
@@ -250,13 +281,191 @@ def convert_webtris_chunks(do_floor=True):
     df = df[['id', 'name', 'lat', 'lon', 'heading', 'time_bin', 'volume', 'speed_counter', 't', 'day']]
     return df
 
-webtris_df = convert_webtris_chunks()
-webtris_df
+# TODO: uncomment if processing data
+# webtris_df = convert_webtris_chunks()
+# webtris_df
+
+
+# +
+# TODO: uncomment if processing data
+# webtris_df['name'] = webtris_df['name'].astype(str)
+# webtris_df.to_parquet(LONDON_PATH / 'speed' / 'webtris_london_201907-202001.parquet', compression="snappy")
```

Comment:
The 4 parquet files produced here have different structure, right? Do we publish the normalization code for the t4c22 competition somewhere?